options.ts

```diff
@@ -31,7 +31,7 @@ var knownOpts: any = {
 	"start": Boolean,
 	"stop": Boolean,
 	"ddi": String, // the path to developer  disk image
-	"print-app-output": Boolean
+	"justlaunch": Boolean
```

Comment:
justLaunch is probably the way to go. I've gone through our nativescript cli options and for the most part we are using camelCase for our options - e.g. availableDevices keyStore*, frameworkPath, forDevice

Two of the few exceptions to this rule I've found are debug-brk and no-client